### DISCUSSION TITLE
Customize CSS

### DISCUSSION THREAD

**Discussion by i7phillip** [dani-garcia/vaultwarden#3549] [Q&A]
Can someone please tell me where to find the css files for vaultwarden? Installed via docker. What I would like is the filters column to scroll itself.

**Comment by BlackDex** [comment]
See https://github.com/dani-garcia/bw_web_builds/releases

Keep in mind that the file names will change every time there is a build done.

**Comment by i7phillip** [comment]
Still not finding CSS files.

**Reply by BlackDex** [reply]
Check the tar.gz file, and within the Vaultwarden docker container check the web-vault folder.

**Reply by i7phillip** [reply]
Thanks. But I already have an instance running on a server. I installed using a docker image. I do not have a web-vault director on my server

**Reply by BlackDex** [reply]
That is within the docker container, like all other files needed to run the server.

**Reply by BlackDex** [reply]
I suggest to also read this https://github.com/dani-garcia/vaultwarden/discussions/2220#discussioncomment-1938492

**Reply by i7phillip** [reply]
Thank you. I'm still a new to this.
